File: src/main/java/com/sudosoo/takeiteasy/dto/comment/CommentResponseDto.java
package com.sudosoo.takeiteasy.dto.comment;

import lombok.AccessLevel;
import lombok.Getter;
import lombok.NoArgsConstructor;

@Getter
@NoArgsConstructor(access = AccessLevel.PRIVATE)
public class CommentResponseDto {

    private Long commentId;
    private String writerName;
    private String content;
    private int likeCount = 0;

    public CommentResponseDto(Long commentId, String writerName, String content, int likeCount) {
        this.commentId = commentId;
        this.writerName = writerName;
        this.content = content;
        this.likeCount = likeCount;
    }

}

File: takeItEasy-application/src/main/kotlin/com/sudosoo/takeItEasy/application/dto/comment/CommentResponseDto.kt
package com.sudosoo.takeItEasy.application.dto.comment

import com.sudosoo.takeItEasy.domain.entity.Comment

class CommentResponseDto(
    val commentId: Long,
    val writer: String,
    val content: String,
    val likeCount: Int
){
    constructor(comment: Comment) : this (
        commentId = comment.id,
        writer = comment.writer,
        content = comment.content,
        likeCount = comment.hearts.size
    )
}
